src/pages/zh-cn/reference/directives-reference.md

```diff
@@ -0,0 +1,242 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: 模板指令参考
+---
+
+**模板指令**是特殊的 HTML 属性，它可以在任一 Astro 组件模板（`.astro` 文件）中使用。
+
+模板指令用某种方式控制元素或组件行为。模板指令可以启用一些编译器功能，使你的生活更轻松（比如使用 `class:list` 而不是 `class`）。指令也可以让 Astro 编译器对该组件进行特殊处理（比如使用 `client:load` 激活组件）。
+
+本页描述了 Astro 中所有可用的模板指令及其工作方式。
+
+## 规则
+
+有效的模板指令需要:
+
+- 在其名称中包括冒号 `:`，使用 `X:Y` 的形式（例如：`client:load`）。
+- 对编译器可见（例如：`<X {...attr}>`，如果 `attr` 包含了指令，则不会工作）。
+
+部分模板指令，可以传递自定义值：
+
+- `<X client:load />`（无法传递值）
+- `<X class:list={['some-css-class']} />`（传递数组）
+
+模板指令永远不会直接包含在组件的最终 HTML 输出中。
+
+## 通用指令
+
+### `class:list`
+
+`class:list={...}` 接收 class 数组，并将其转换为 class 字符串。这是受流行的 @lukeed [clsx](https://github.com/lukeed/clsx)辅助库的启发，但直接内置在 Astro 里。
+
+`class:list` 接收数组，其中有几种不同的可能值：
+
+- `string`：添加到 `class` 元素
+- `Object`：添加到键值对到 `class` 元素
+- `Array`：扁平化
+- `Set`：扁平化
+
+```astro
+<!-- 原先 -->
+<span class:list={[ 'hello goodbye', { hello: true, world: true }, new Set([ 'hello', 'friend' ]) ]} />
+<!-- 输出 -->
+<span class="hello goodbye world friend"></span>
+```
+
+它会自动删除重复值。
+
+### `set:html`
+
+`set:html={string}` 将 HTML 字符串注入元素中，类似于设置 `el.innerHTML`。
+
+**该值不会被 Astro 自动转义！**请确保你信任该值，或者在传递给模板前对其进行手动转义。忘记这样做可能会使你受到[跨网站脚本（XSS）攻击](https://owasp.org/www-community/attacks/xss/)。
```

Comment:
Looks like the character "！" is somehow making the text not bold here? Maybe using the "!" character instead can help (if there isn't a better solution).